crates/oxc_linter/src/rules/eslint/no_unused_vars/options.rs

```diff
@@ -657,7 +659,7 @@ mod tests {
         .unwrap();
         // option object provided, no default varsIgnorePattern
         assert!(rule.vars_ignore_pattern.is_none());
-        assert!(rule.args_ignore_pattern.unwrap().as_str() == "^_");
+        assert!(matches!(rule.args_ignore_pattern, IgnorePattern::Default));
```

Comment:
Can use is_default in this file instead:
```suggestion
        assert!(rule.args_ignore_pattern.is_default());
```